src/main/resources/anniv5Resources/localization/eng/madsciencepack/Cardstrings.json

```diff
@@ -17,30 +17,30 @@
   },
   "${ModID}:Attach": {
     "NAME": "Attach",
-    "DESCRIPTION": "Choose a non-Power in hand to ${ModID}:Absorb into a discarded card, then ${ModID}:Modify it: \"Play the *Absorbed card\". NL Exhaust.",
```

Comment:
Oh, seems like you removed all your updates to the strings with this. Seems like you fucked up.

Reply:
Faaaak been doing fine with merge conflicts up till now.